me.njean.shootinpactris.json

```diff
@@ -0,0 +1,89 @@
+{
+    "app-id": "me.njean.shootingpactris",
+    "branch": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "shpactris",
+    "finish-args": [
+                        "--socket=x11",
+                        "--share=network",
+                        "--device=all",
+                        "--socket=pulseaudio"
+                    ],
+    "modules": [
+        "shared-modules/glu/glu-9.json",
```

Comment:
You forgot to add shared-modules a git submodule

Reply:
Oups. Added!